var/spack/repos/builtin/packages/conduit/package.py

```diff
@@ -30,7 +37,7 @@ class Conduit(Package):
     url      = "https://github.com/LLNL/conduit/releases/download/v0.3.0/conduit-v0.3.0-src-with-blt.tar.gz"
     git      = "https://github.com/LLNL/conduit.git"
 
-    version('master', branch='master', submodules=True)
+    version('master', branch='master', submodules=True, preferred=True)
```

Comment:
I don't think an unstable branch should ever be the default over a stable release.

Reply:
I am planning on an new release for conduit, 0.3.1 is almost two years old. 
That said, these packages require master -- and most consumers are using master.